tests/acceptance/check-deleted-suites-in-expected-failure.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+# when deleting the tests suites from /features there might be the tests scenarios that might be in the expected to failure file
+# this script checks if there are such scenarios in the expected to failure file which needs to be deleted
+
+# helper functions
+log_error() {
+	echo -e "\e[31m$1\e[0m"
+}
+
+log_info() {
+	echo -e "\e[37m$1\e[0m"
+}
+
+log_success() {
+	echo -e "\e[32m$1\e[0m"
+}
+
+SCRIPT_PATH=$(dirname "$0")
+PATH_TO_SUITES="${SCRIPT_PATH}/features"
+EXPECTED_FAILURE_FILES=("expected-failures-localAPI-on-OCIS-storage.md" "expected-failures-API-on-OCIS-storage.md")
+# contains all the suites names inside tests/acceptance/features
+AVAILABLE_SUITES=($(ls -l "$PATH_TO_SUITES" | grep '^d' | awk '{print $NF}'))
+
+# regex to match [someSuites/someFeatureFile.feature:lineNumber]
+SCENARIO_REGEX="\[([a-zA-Z0-9]+/[a-zA-Z0-9]+\.feature:[0-9]+)]"
+
+# contains all those suites available in the expected to failure files in pattern [someSuites/someFeatureFile.feature:lineNumber]
+EXPECTED_FAILURE_SCENARIOS=()
+for expected_failure_file in "${EXPECTED_FAILURE_FILES[@]}"; do
```

Comment:
how about enclosing all login within this loop?

Reply:
would be good. Thanks!